### PR TITLE
Add energy cutoff for plane wave and dual basis Hamiltonians.

### PR DESCRIPTION
**Pull request by Spaceenter**

### CODE REVIEW

**Comment by Spaceenter** [comment]
@babbush @jarrodmcc

File: src/fermilib/utils/_jellium.py [hunk 1]
```diff
@@ -411,31 +461,29 @@ def jellium_model(n_dimensions, grid_length, length_scale,
...
+        hamiltonian = position_kinetic_operator(n_dimensions, grid_length,
+                                                length_scale, spinless,
+                                                energy_cutoff)
+        hamiltonian += position_potential_operator(n_dimensions, grid_length,
+                                                   length_scale, spinless,
+                                                   energy_cutoff)
     return hamiltonian
 
 
+# TODO(wsws): Update with the latest formula, add U term, and add energy cut.
```
Comment (babbush) [inline src/fermilib/utils/_jellium.py]: I would normally be okay with merging changes that have this kind of TODO into develop but I think we might make a push to master tonight (Sunday) or tomorrow and I don't want to have this TODO end up in master. So I will either wait until we 1) push from develop to master or 2) you complete this TODO before merging this pull request. Does that make sense?

File: src/fermilib/utils/_plane_wave_hamiltonian.py [hunk 2]
```diff
@@ -91,29 +95,39 @@ def plane_wave_u_operator(n_dimensions, grid_length, length_scale, geometry,
...
         spinless: Bool, whether to use the spinless model or not.
+        energy_cutoff: Float, energy cutoff.
 
     Returns:
         operator: An instance of the FermionOperator class.
     """
     n_points = grid_length ** n_dimensions
     volume = length_scale ** float(n_dimensions)
     prefactor = -4.0 * numpy.pi / volume
-    operator = None
```
Comment (babbush) [inline src/fermilib/utils/_plane_wave_hamiltonian.py]: Good change.

File: src/fermilib/utils/_plane_wave_hamiltonian.py [hunk 3]
```diff
@@ -164,14 +177,14 @@ def plane_wave_hamiltonian(n_dimensions, grid_length, length_scale, geometry,
 
     if momentum_space:
         return jellium_model(n_dimensions, grid_length, length_scale, spinless,
-                             True) + \
+                             True, energy_cutoff) + \
```
Comment (babbush) [inline src/fermilib/utils/_plane_wave_hamiltonian.py]: Please see Issue #16 opened by @damiansteiger . We are trying get rid of all line conjugation with the "\" symbol. Would you mind removing these from this file prior to merge?

**Pull request closed without merging**